src/rules/speech_control.ts

```diff
@@ -583,20 +588,34 @@ export function initRules_bc_speech_control() {
 				if (isObject(data) &&
 					!data.BeepType &&
 					typeof data.MemberNumber === "number" &&
-					state.isEnforced &&
+					state.inEffect &&
 					state.customData &&
 					!state.customData.whitelistedMemberNumbers.includes(data.MemberNumber) &&
 					(!Player.CanInteract() || !state.customData.onlyWhenBound)
 				) {
-					if (state.customData.autoreplyText) {
+					if (state.customData.autoreplyText && !data.Message?.startsWith("[Automatic reply by BCX]\n")) {
```

Comment:
Here you want to also check the message actually is a string - there can be many different things in a message.

Reply:
Fixed